sdk/python/tests/compiler/testdata/sequential_noninlined.yaml

```diff
@@ -74,7 +85,7 @@ spec:
         steps:
         - name: main
           args:
-          - echo "$0"
+          - echo
```

Comment:
quick question, does this command still work without "$0"?

Reply:
yes, for the old one, it uses parameter substitution. the new one just two arguments. both produce the same output